test/fixtures/ast/error-message-outside/ast.json

```diff
@@ -1345,9 +1345,9 @@
             },
             {
                 "message": "Unexpected end of expression.",
-                "index": 76,
+                "index": 75,
```

Comment:
This appears to be the accurate location data for the trailing `1` in the expression `abc + (1` in `test/fixtures/ast/error-message-outside/source.vue`